src/Cosmos/Serialization.cpp

```diff
@@ -20,6 +20,16 @@ const static std::string AMINO_PREFIX_SEND_COIN_MESSAGE = "cosmos-sdk/MsgSend";
 const static std::string AMINO_PREFIX_STAKE_MESSAGE = "cosmos-sdk/MsgDelegate";
 const static std::string AMINO_PREFIX_TRANSACTION = "auth/StdTx";
 const static std::string AMINO_PREFIX_PUBLIC_KEY = "tendermint/PubKeySecp256k1";
+const static std::string BROADCAST_MODE = "async";
+
+json higherWrapperJSON(const std::string& broadCastMode, json& jsonObj) {
+    json jsonMsgWrapper;
+
+    jsonMsgWrapper["tx"] = jsonObj;
+    jsonMsgWrapper["mode"] = broadCastMode;
```

Comment:
I think we should expose `broadcastMode ` so the caller can specify which format he would like